glutin/src/context.rs

```diff
@@ -55,10 +55,10 @@ pub trait NotCurrentGlContext: Sealed {
     ///
     /// # Platform specific
     ///
-    /// **macOS:** - **This will block if your main thread is blocked.**
-    /// **Wayland:** - This call may latch the underlying back buffer (will do
-    /// with mesa drivers), meaning that all resize operations will apply after
-    /// the next [`GlSurface::swap_buffers`].
+    /// - **macOS: this will block if your main thread is blocked;**
```

Comment:
Nit: this feels more natural (same below) but not sure if it's worth it.

```suggestion
    /// - **macOS: this will block if your main thread is blocked**;
```